firmware/library/L1_Drivers/button_driver.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <cstdint>
+#include <cstdio>
+#include "L0_LowLevel/LPC40xx.h"
+#include "L1_Drivers/pin_configure.hpp"
```

Comment:
None of these includes are being used in this file. Please remove them. Only include stuff you are using.